src/app/zap-templates/templates/app/CHIPClientCallbacks-src.zapt

```diff
@@ -183,6 +183,24 @@ void LogStatus(uint8_t status)
     }
 }
 
+void LogStringAttribute(const uint8_t * string, const uint16_t length, const bool isAscii)
+{
+    if (isAscii)
+    {
+        ChipLogProgress(Zcl, "  value: %.*s", length, string);
+        return;
+    }
+
+    char buffer[CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE] = "  value: ";
+    char * bufferPos                              = buffer + strlen(buffer);
+    char * const bufferEnd                        = buffer + CHIP_CONFIG_LOG_MESSAGE_MAX_SIZE;
+
+    for (uint16_t i = 0; i < length && bufferPos + 3 < bufferEnd; i++, bufferPos += 3)
+        snprintf(bufferPos, bufferEnd - bufferPos, "%02X ", string[i]);
```

Comment:
Please add brackets.

Is there a constant somewhere we can use instead of `3`. Reading at the code it is unclear to me why we use `3`

Reply:
Is there a written or unwritten rule about using brackets for one-line statements? I usually don't use them as I prefer shorter code and I've heard no complaints so far :)
I'll apply your suggestions though.